### PR TITLE
df: fix too low values in I* columns

### LINKED ISSUE #3349: df: values in the "Inodes", "IUsed", and "IFree" columns are too low
The values reported in the `Inodes`, `IUsed`, and `IFree` columns are by a factor of 1024 lower than those reported by GNU df.

```
$ ./target/debug/coreutils df --output=source,itotal,iused,iavail
Filesystem             Inodes        IUsed        IFree 
dev                       978            0          977 
run                       980            0          979 
/dev/sda8               28416          966        27449 
tmpfs                     980            0          980 
tmpfs                    1024            0         1023 
/dev/sda2                   0            0            0 
tmpfs                     196            0          195

$ df --output=source,itotal,iused,iavail
Filesystem       Inodes  IUsed    IFree
dev             1001714    567  1001147
run             1003691    878  1002813
/dev/sda8      29097984 989575 28108409
tmpfs           1003691    158  1003533
tmpfs           1048576     43  1048533
/dev/sda2             0      0        0
tmpfs            200738     81   200657
```

### PR DESCRIPTION
**Pull request by cakebaker**
Instead of using a single `scaled()` function for both inodes and bytes, I introduced two new specialized functions.

Fixes #3349.

### CODE REVIEW

File: src/uu/df/src/table.rs [hunk 1]
```diff
@@ -212,25 +212,57 @@ impl<'a> DisplayRow<'a> {
         Self { row, options }
     }
 
-    /// Get a string giving the scaled version of the input number.
+    /// Get a human readable string giving the scaled version of the input number.
     ///
     /// The scaling factor is defined in the `options` field.
     ///
+    /// This function is meant to be used by scaled_bytes() resp. scaled_inodes() only.
```
Comment (jfinkels) [inline src/uu/df/src/table.rs:219]: I don't understand the use of "resp." in this sentence. Could you rephrase it?

**Review by jfinkels — APPROVED** [review]

File: src/uu/df/src/table.rs (hunk 1 above)
Comment (cakebaker) [reply src/uu/df/src/table.rs]: Ok, I rephrased it. I hope it's an improvement.
